user_class/spec/user_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative '../user'
+require 'factories'
+
+RSpec.describe User do
+  subject(:users) { described_class.new }
```

Comment:
Prefer annonymous subject than named.
```suggestion
  subject { described_class.new }
```